AppCenter/AppCenter/MSACCustomProperties.h

```diff
@@ -10,6 +10,7 @@
  * Custom properties builder.
  * Collects multiple properties to send in one log.
  */
+NS_SWIFT_NAME(CustomProperties)
```

Comment:
Please make sure that

```objc
- (instancetype)clearPropertyForKey:(NSString *)key;
```

converted to

```swift
func clearProperty(for key: String)
```

Reply:
Addressed in ff4e86257ac5d4d7184065423142be07d222683e